GPT/gpt.py

```diff
@@ -212,72 +207,75 @@ def gpt_insert_response(
             # Skip inserting the response if the user is just viewing the thread in the window
             actions.user.confirmation_gui_refresh_thread()
 
+        message_text_no_images = extract_message(gpt_message)
```

Comment:
Extract message currently returns text so if there's an image here it will return an empty string